app/models/tag.rb

```diff
@@ -1,4 +1,8 @@
 class Tag < ApplicationRecord
   has_many :jobs_tags
   has_many :jobs, through: :jobs_tags
+
+  def count
+    self.jobs.count
+  end
```

Comment:
Great method here. I'd argue that your test for this method should live within a model spec vs your feature spec, though.